pages/add/form/auto-complete-input.js

```diff
@@ -159,12 +159,11 @@ export default class AutoCompleteInput extends Component {
           tags={this.state.data}
           suggestions={this.getFilteredSuggestions()}
           allowNew={true}
-          autofocus={false}
```

Comment:
This is no longer supported.

Reply:
It took me a while to convince them, but I'm glad they finally removed it.

Reply:
Yay!